nextflow.config

```
@@ -252,7 +252,7 @@ manifest {
     description     = """Pipeline to analyse RNAseq from raredisease patients"""
     mainScript      = 'main.nf'
     nextflowVersion = '!>=23.04.0'
-    version         = '1.0dev'
+    version = '0.1.0'
```

```suggestion
    version = '1.0.0'
```
Let's go for version 1 :) We can still update it